src/core_plugins/console/public/src/es.js

```diff
@@ -15,11 +15,11 @@ export function send(method, path, data) {
     method = "POST";
   }
 
-  let contentType;
+  // even body-less requests will be sent with JSON as the content-type to appease Kibana
+  let contentType = 'application/json';
```

Comment:
It looks like there is an open PR that trying to fix the same thing as well: https://github.com/elastic/kibana/pull/12294.

Sorry for wasting your time @pickypg :( but yesterday we decided to rollback "content-type as xsrf token" PR for 6.0 so that we can have more time to decide how to be fully compliant with ES and to not release half-baked solution for such important thing.

I still think it will be good to change our code to always pass correct content-type though, but at this point we can't get rid of old xsrf approach. What are your thoughts @epixa ?